### DISCUSSION TITLE
will cardinal move to agplv3?

### DISCUSSION THREAD

**Discussion by RustoMCSpit** [DISTRHO/Cardinal#660] [General]
just was thinking

**Comment by dromer** [comment]
Because we would want to prevent "Cardinal as a service"?

Highly improbable as likely all the "Non-Commercial" clauses in the art-work would prevent this (at least when using the UI).

Investigating/considering this would be extremely low priority and very unlikely to happen.

**Comment by RustoMCSpit** [comment]
> Because we would want to prevent "Cardinal as a service"?

huh? agplv3 is stronger the gpl3, not following

> Highly improbable as likely all the "Non-Commercial" clauses in the art-work would prevent this (at least when using the UI).

it isnt commerical.

**Reply by dromer** [reply]
Do you understand why the AGPL exists?

Otherwise: what is the point of re-licensing?

**Reply by RustoMCSpit** [reply]
agpl is to prevent a serverside loophole

**Reply by falkTX** [reply]
it is not a loophole if it is intended. GPLv2 also allows for "tivoization" which is fine with some projects.

for the case of cardinal the license is always going to be gplv3+
